src/components/railBottom/controls/SubmenuPopper/SubmenuPopper.vue

```diff
@@ -1,45 +1,94 @@
 <template>
   <popper
-    trigger="clickToToggle"
+    v-if="items"
+    trigger="clickToOpen"
     :options="{
       placement: 'top',
       modifiers: { offset: { offset: '0,20px' } }
     }"
     :visible-arrow=false
+    :force-show="popperType === data.type"
   >
-    <div v-if="data.items">
+    <div>
       <ul class="control-submenu">
-        <li v-for="(item, key) in data.items" class="control-submenu-list"  :key="key">
-          <img :src=item.iconSrc :alt=item.label>
-          <div>
+        <li v-for="(item, key) in items"
+          :class="{'control-submenu-list active': selectedItem === item.type, 'control-submenu-list': selectedItem !== item.type}"
+          :key="key"
+          @click="onClickHandler($event, item)"
+        >
+          <inline-svg :src="item.icon" :alt=item.label />
+          <div class="control-submenu-list-label">
             {{ item.label }}
           </div>
         </li>
       </ul>
     </div>
-    <div slot="reference">
+    <a
+      class="control-submenu-item "
+      slot="reference"
+      :alt=data.label
+      :title="$t(data.label)"
+      v-b-tooltip.hover
+    >
       <div class="control-submenu-options">
         <span />
       </div>
-      <img
-        :src=data.iconSrc
-        :alt=data.label
-        :title="$t(data.label)"
-        v-b-tooltip.hover
-      >
-    </div>
+
+      <inline-svg :src="data.icon" />
+    </a>
   </popper>
+  <a v-else class="control-submenu-item"
+    :alt=data.label
+    :title="$t(data.label)"
+    v-b-tooltip.hover
+  >
+    <inline-svg :src=data.icon />
+  </a>
 </template>
 
 <script>
+import { BOTTOM } from '@/components/controls/rankConstants';
 import Popper from 'vue-popperjs';
 import 'vue-popperjs/dist/vue-popper.css';
+import InlineSvg from 'vue-inline-svg';
+
 export default ({
   props: {
     data: { type: Object },
+    selectedItem: { type: String },
+    popperType: { type: String },
   },
   components: {
     Popper,
+    InlineSvg,
+  },
+  data() {
+    return {
+      wasClickedSubmenu: false,
+      element: null,
+      wasClicked: false,
+    };
+  },
+  computed: {
+    items() {
+      return  this.data.items && this.data.items
+        .filter(item => item.control)
+        .map(item => ({
+          type: item.id,
+          icon: item.icon,
+          label: item.label,
+          bpmnType: item.bpmnType,
+          rank: item.rank || BOTTOM,
+          items: item.items || null,
+        }))
+        .sort((node1, node2) => node1.rank - node2.rank);
+    },
```

Comment:
same comment about getting this from Vuex

Reply:
It will be applied